yocto/recipes-browser/cog/cog_0.12.%.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+SRC_URI += "file://cog-set-opaque-region_v0.12.patch"
```

Comment:
Does this include cog 0.12.1?  I was doing some work for the performance-report and found a bug in cog that it's solved in 0.12.1, so I think that it would be better to bump the version at least to 0.12.1, but not sure how's done.

Reply:
This patch needs updating to make it apply cleanly, I think. Even better: why not submitting it to upstream Cog, so there will be no longer the need of carrying it as part of the Bitbake recipe, and everybody could benefit from it? 😉